src/main/java/com/aws/greengrass/util/platforms/windows/WindowsExec.java

```diff
@@ -140,6 +144,62 @@ public synchronized void close() throws IOException {
             return;
         }
 
+        try {
+            stopGracefully();
+        } finally {
+            stopForcefully();
+        }
+    }
+
+    private void stopGracefully() {
+        int pid = Processes.newPidProcess(process).getPid();
+        // Global lock since we're messing with a shared resource (the console)
+        boolean sentConsoleCtrlEvent = false;
+        synchronized (Kernel32Ex.INSTANCE) {
```

Comment:
Please verify that after attaching and then freeing console, that _our_ application is still able to receive ctrl+c on windows. Just spawn a process during Nucleus launch and graceful kill it; then verify that ctrl+c still works. 
I am wondering if we need to re-attach at the end to our own PID's console or not.